normative-conventions.md

```diff
@@ -37,3 +37,11 @@ For the purposes of this guideline `-0` is considered to be an integral number (
 Some APIs intentionally round non-integral inputs, for example as an attempt to provide a best-effort behavior, rather than because the API fundamentally only makes sense with integers. This guideline does not apply to those cases.
 
 NB: This convention is new as of 2024, and most earlier parts of the language do not follow it.
+
+## Reject primitive Strings in iterable-taking APIs and syntax
+
+Any time an iterable value (a value that has a `Symbol.iterator` method) is expected, primitive Strings should be treated as if they were not iterable. Usually, this will mean throwing a `TypeError`. String Objects, however, should be treated like any other Object.
```

Comment:
```suggestion
Any time an iterable value (a value that has a `Symbol.iterator` method) is expected, primitive Strings should be treated as if they were not iterable. Usually, this will mean throwing a `TypeError` when the API is passed any non-`Object` value. String Objects, however, should be treated like any other Object.
```

Reply:
Is the rule to reject all primitives and just never ToObject before looking for the symbol?

Reply:
Yeah. That means that if someone adds `Number.prototye[Symbol.iterator]` it won't get iterated. Which is easier to learn - we're not carving out Strings specifically, but rather primitives.

Reply:
Okay I'm fine with that. I'll re-write it.